meta-mender-octavo-osd32mp/conf/layer.conf

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 Northern.tech AS
```

Comment:
This should be 2023, and your copyright.

Reply:
Ahh, right. 
Changed it to `# Copyright 2023 António Oliveira (Konsulko Group)`
I hope this is suitable. If not please let me know what is more appropriate (I'm rather inexperienced at this kind of legalese stuff)